docs/how_tos/theming.md

```diff
@@ -0,0 +1,76 @@
+# Theming support with Paragon
+
+This document serves as a guide to using `@edx/frontend-platform` to support MFE theming with Paragon using theme CSS loaded externally (e.g., from a CDN). By serving CSS loaded externally, consuming applications of Paragon no longer need to be responsible for compiling the theme SCSS to CSS themselves and instead use a pre-compiled CSS file. In doing so, this allows making changes to the Paragon theme without needing to necessarily re-build and re-deploy all consuming applications. We would also get a meaningful gain in performance as loading the compiled theme CSS from an external CDN means micro-frontends (MFEs) can include cached styles instead of needing to load essentially duplicate theme styles as users navigate across different MFEs.
+
+## Overview
+
+![overview of paragon theme loader](./assets/paragon-theme-loader.png "Paragon theme loader")
+
+## Theme URL configuration
+
+Paragon supports 2 mechanisms for configuring the Paragon theme URLs:
+* JavaScript-based configuration via `env.config.js`.
+* MFE runtime configuration API via `edx-platform`
+
+Using either configuration mechanism, a `PARAGON_THEME_URLS` configuration setting must be created to point to the externally hosted Paragon theme CSS files, e.g.:
+
+```js
+const config = {
+    PARAGON_THEME_URLS = {
+        core: 'https://cdn.jsdelivr.net/npm/@edx/paragon@$paragonVersion/dist/core.css',
+        variants: {
+            light: 'https://cdn.jsdelivr.net/npm/@edx/paragon@$paragonVersion/dist/light.css',
+        },
+    },
+};
+export default config;
+```
+
+### JavaScript-based configuration
+
+Another approach to configuration with `@edx/frontend-platform` is to create a `env.config.js` file in the root of the repository, similar to the environment variable configuration mentioned above. However, in this case, the configuration is defined as a JavaScript file, which affords consumers to use more complex data types than just strings as in the environment variable approach.
```

Comment:
Note, I don't think the environment variable configuration mentioned above is actually mentioned above anymore. :) This paragraph may need to be revisited.